test/built-ins/decodeURI/S15.1.3.1_A2.2_T1.js

```diff
@@ -15,17 +16,16 @@ var uriReserved = [";", "/", "?", ":", "@", "&", "=", "+", "$", ","];
 l:
 for (var indexB1 = 0x00; indexB1 <= 0x7F; indexB1++) {       
   count++;
-  var hexB1 = decimalToHexString(indexB1);  
+  var hexB1 = decimalToPercentHexString(indexB1);
   var index = indexB1;  
   try {
     var hex = String.fromCharCode(index);
     for (var indexC = 0; indexC < uriReserved.length; indexC++) {
       if (hex === uriReserved[indexC]) continue l;        
     } 
     if (hex === "#") continue l;
-    if (decodeURI("%" + hexB1.substring(2)) === hex) continue;
+    if (decodeURI(hexB1) === hex) continue;
   } catch (e) {
-    if (e instanceof Test262Error) throw e;
   }   
```

Comment:
I believe we should remove the whole try/catch wrapping. If it throws, let the test fail.